src/driver/main.cpp

```diff
@@ -978,6 +1001,14 @@ int main(int argc, const char* argv[], const char* envp[])
         for(auto&& e : unused_envs)
             std::cout << "Unused environment variable: " << e << "\n";
 
+        // Print end timestamp
+        auto end_time = std::chrono::system_clock::now();
+        std::cout << "End time: " << get_formatted_timestamp(end_time) << std::endl;
+
+        // Print total duration
+        auto [seconds, milliseconds] = get_clock_time_from_duration(end_time - start_time);
+        std::cout << "Time: " << seconds << "." << milliseconds << "s" << std::endl;
```

Comment:
Add the duration to the complete line: `std::cout << "[ " << get_version() << " ] Complete(" << seconds << "s): " <<  driver_invocation << std::endl`.